cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -1573,6 +1573,12 @@
 
 ;; UImm5 Helpers
 
+;; Extractor that matches a `Value` equivalent to a replicated UImm5 on all lanes.
+;; TODO: Try matching vconst here as well
```

Comment:
Can you either resolve this TODO in this PR or turn it into `TODO(#1234)` with a reference to a follow up issue?